### PR TITLE
[Elixir] Add automated comment for take-a-number

### PR DESCRIPTION
**Pull request by angelikatyborska**
Related PR: https://github.com/exercism/elixir-analyzer/pull/33

### CODE REVIEW

File: automated-comments/elixir/take-a-number/do_not_use_abstractions.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The purpose of this exercise is teach the basics of processes and messages. Please solve it without using abstractions such as the `Agent` module or the `GenServer` behaviour.
```
Comment (SleeplessByte) [inline automated-comments/elixir/take-a-number/do_not_use_abstractions.md:1]:
```suggestion
The purpose of this exercise is to teach the basics of processes and messages. Solve it without using abstractions such as the `Agent` module or the `GenServer` behaviour. 
```

**Review by SleeplessByte — DISMISSED** [review]
Merge if you feel like it 👍🏽

**Comment by angelikatyborska** [comment]
I personally cannot merge due to "At least 1 approving review is required by reviewers with write access." 🤷

**Review by SleeplessByte — APPROVED** [review]
It was supposed to be an approve!

**Pull request merged**